openapi/ebay.listing/client.bal

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import  ballerina/http;
+
+# Client configuration
+#
+# + authConfig - OAuth client configuration  
+# + secureSocketConfig - SSH configuration  
+public type ClientConfig record {
+    http:BearerTokenConfig|http:OAuth2RefreshTokenGrantConfig authConfig;
+    http:ClientSecureSocket secureSocketConfig?;
+};
+
+# This API is used by partners to create draft listings on eBay for their seller.
+# 
+# + clientEp - Connector http endpoint
+public client class Client {
+
+    http:Client clientEp;
+    # Client initialization.
+    #
+    # + clientConfig - OAuth client configuration
+    # + serviceUrl - Service URL
+    # + return - Returns error at failure of client initialization
+    public isolated function init(ClientConfig clientConfig, string serviceUrl = "https://api.ebay.com/sell/listing/v1_beta") returns error? {
+        http:ClientSecureSocket? secureSocketConfig = clientConfig?.secureSocketConfig;
+        http:Client httpEp = check new (serviceUrl, { auth: clientConfig.authConfig, secureSocket: secureSocketConfig });
+        self.clientEp = httpEp;
+    }
```

Comment:
We better have an empty line between init() {..} and the doc comment start of the next operation. @aneeshafedo

Reply:
In the current code generation there is no empty line between each function not only in the init function. Since the code is written directly from the ballerina syntax tree need to look into the capability of facilitating this kind of formatting issues.

Reply:
Yes. Let's fix this as an separate effort to format the generated code.